tests/nlu/test_evaluation.py

```diff
@@ -315,7 +302,7 @@ def test_run_cv_evaluation():
     assert len(entity_results.test["CRFEntityExtractor"]["F1-score"]) == n_folds
 
 
-def test_run_cv_evaluation_with_response_selector():
+def test_run_cv_evaluation_with_response_selector(supervised_embeddings_config):
```

Comment:
supervised_embeddings_config doesn't have response selector component in it

Reply:
the config it not used, will remove it